src/externalModulesES6modules.js

```diff
@@ -4,4 +4,15 @@ import * as cornerstoneMath from '../../cornerstoneMath/src/index.js';
 const $ = window.$;
 const Hammer = window.Hammer;
 
-export { $, Hammer, cornerstone, cornerstoneMath };
+let cornerstone;
```

Comment:
are you sure we can't do:
````
let cornerstone = window.cornerstone; 
````

here? Then we wouldn't have to change examples where things are added via the script tag? Not sure if this would cause confusion, though. I just don't want people to have to change code that already works.